app/models/forms/affiliation.rb

```diff
@@ -0,0 +1,41 @@
+module Forms
```

Comment:
Yes, it probably makes sense to put this in `app/forms`. However, since we already have our reform forms in this location, I put them in `app/models/forms`.